client/src/components/Management.jsx

```diff
@@ -0,0 +1,5 @@
+function Management() {
```

Comment:
I'm guessing this component was just something that you forgot to remove. If thats the case please remove it.